src/libs/actions/Report.js

```diff
@@ -963,6 +963,16 @@ function deleteReportComment(reportID, reportAction) {
         optimisticData.push(optimisticParentReportData);
     }
 
+    // If the reportID doesn't equal the originalReportID then this is the first comment on a thread report and we need to trigger
+    // some sort of update to let the LHN know that the parentReportAction is now deleted.
+    if (reportID !== originalReportID) {
```

Comment:
Would this make sense here?
```suggestion
    if (isThreadFirstChat(reportAction, reportID)) {
```

Reply:
I think both are true, but yeah - that's a bit clearer though - definitely prefer it.